test_scripts/Smoke/Registration/ATF_Register_5_connection.lua

```diff
@@ -104,30 +96,23 @@ end
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 
-function Test:FirstConnection()
-  createConnectionAndRegisterApp(self, device1, "mobile1.out", default_app_params1)
-end
-
-function Test:SecondConnection()
-  createConnectionAndRegisterApp(self, device2, "mobile2.out", default_app_params2)
-end
-
-function Test:ThirdConnection()
-  createConnectionAndRegisterApp(self, device3, "mobile3.out", default_app_params3)
+for i = 1, 5 do
+  function Test:FirstConnection()
+    local filename = "mobile" .. i .. ".out"
+    local app = config["application"..i].registerAppInterfaceParams
+    createConnectionAndRegisterApp(self, devices[i], filename, app)
+  end
 end
-
-function Test:FourthConnection()
-  createConnectionAndRegisterApp(self, device4, "mobile4.out", default_app_params4)  
-end
-
-function Test:FifthConnection()
-  createConnectionAndRegisterApp(self, device5, "mobile5.out", default_app_params5)  
-end
-
 -- [[ Postconditions ]]
 commonFunctions:newTestCasesGroup("Postcondition")
 function Test.Stop_SDL()
   StopSDL()
 end
 
+function Test.ShutDownDummyConnections()
+  for i = 2, 5 do
```

Comment:
@dboltovskyi Why was used next loop:
```
for i = 2, 5 do
   os.execute("ifconfig lo:" .. i-1 .." down")
end
```
instead of
```
for i = 1, 4 do
   os.execute("ifconfig lo:" .. i .." down")
end
``` 
?